### PR TITLE
Fixed and Improved Brazil's IBGE sources

### PR DESCRIPTION
**Pull request by matheusgomesms**
Fixed issue #983 (updated all URLs when possible), added privacy policies, and did minor fixes on them, as discussed previously on PR #1233.

### CODE REVIEW

**Comment by matheusgomesms** [comment]
It seems all geometries are not good (only error), but given it covers the whole country, I'm not confortable with expanding the geometries, covering also parts of neighbor countries.

**Comment by rbuffat** [comment]
@matheusgomesms Don't worry about the geometry checks. The threshold is arbitrary chosen. It works for many cases, but here I think they can be ignored.

File: sources/south-america/br/IBGE_Street_Names.geojson [hunk 1]
```diff
@@ -2,13 +2,14 @@
...
         "attribution": {
+            "url": "https://www.ibge.gov.br/",
             "text": "IBGE, OSM Brasil"
         },
         "country_code": "BR",
         "icon": "https://osmlab.github.io/editor-layer-index/sources/south-america/br/IBGE.png",
         "id": "IBGE_Nomes_Ruas_2019",
         "license_url": "https://wiki.openstreetmap.org/wiki/IBGE_Street_Names",
-        "privacy_policy_url": "https://www.mapbox.com/legal/privacy",
+        "privacy_policy_url": "https://www.ibge.gov.br/np_download/novoportal/documentos_institucionais/PoliticaPrivacidade.pdf",
```
Comment (andrewharvey) [inline sources/south-america/br/IBGE_Street_Names.geojson]: It should be Mapbox's privacy policy as the url is api.mapbox.com.

File: sources/south-america/br/IBGE_DF_Adresses.geojson [hunk 2]
```diff
@@ -2,17 +2,20 @@
     "type": "Feature",
     "properties": {
         "id": "IBGE_DF_Addresses",
-        "name": "IBGE Distrito Federal",
+        "name": "IBGE Endereços Distrito Federal",
         "overlay": true,
         "url": "https://{switch:a,b,c,d}.tiles.mapbox.com/styles/v1/wille/cirnnxni1000jg8nfppc8g7pm/tiles/256/{zoom}/{x}/{y}?access_token=pk.eyJ1Ijoid2lsbGUiLCJhIjoiY2t1dm4xYmF2NmEzdDMycTY3MjJtNmRwMiJ9._3qoS2hLLiJEkIDsVO4DoA",
-        "privacy_policy_url": "https://www.mapbox.com/legal/privacy",
+        "privacy_policy_url": "https://www.ibge.gov.br/np_download/novoportal/documentos_institucionais/PoliticaPrivacidade.pdf",
```
Comment (andrewharvey) [inline sources/south-america/br/IBGE_DF_Adresses.geojson]: It should be Mapbox's privacy policy as the url is mapbox.com

**Comment by andrewharvey** [comment]
Thanks this looks good to me, just two minor issues I found, if you wanted to rebase/resolve conflicts then we can merge.

**Comment by matheusgomesms** [comment]
Great, I just updated again the policies as suggested, thanks!

**Comment by andrewharvey** [comment]
Thanks, GitHub is reporting conflicts which need resolving before this can be merged.

**Comment by matheusgomesms** [comment]
Hi @andrewharvey

I guess it was updated here but not on my branch, but I finally resolved the conflicts. Also I updated all files again, given that we have more updated data (2019->2021).

I guess we are finally good to go!

If this is merged, we can also close issue #983

**Pull request merged**